static/js/components/search/NestedAggregatingMenuFilter.js

```diff
@@ -0,0 +1,292 @@
+import PatchedMenuFilter from './PatchedMenuFilter';
```

Comment:
I just merged a PR which fixes tests in `LearnerSearchPage_test` and also adds a few tests for the work history and education facets. Can we add a test to make sure the query is being constructed properly?